propka/lib.py

```diff
@@ -171,6 +171,8 @@ def loadOptions(*args):
            help="not activated yet")
     parser.add_option("--verbosity",  dest="verbosity", action="store_const",
            help="level of printout - not activated yet")
+    parser.add_option("--no-print",  dest="no_print", action="store_true", default=False,
```

Comment:
Instead of adding a new option (--no-print), how about we get the existing --verbosity option to work?   If set to "none" it would be equivalent of --no-print.  In the future we could add more functionality - such as only printing out errors/warnings.